server/config.go

```diff
@@ -176,6 +182,18 @@ func (c *Config) Parse(arguments []string) error {
 		if err != nil {
 			return errors.Trace(err)
 		}
+
+		// Backward compatibility for toml config
+		if c.LogFileDeprecated != "" && c.Log.Filename == "" {
+			c.Log.Filename = c.LogFileDeprecated
+			msg := fmt.Sprintf("log-file in %s is deprecated, use [log] instead", c.configFile)
+			c.WarningMsgs = append(c.WarningMsgs, msg)
```

Comment:
Current implementation is OK for future planning.
`config.Parse()`, `logutil.InitLogger()` may all update warning messages.